aio_pika/connection.py

```diff
@@ -394,6 +394,7 @@ def connect(url: str=None, *, host: str='localhost',
         login = url.user or login
         password = url.password or password
         virtualhost = url.path[1:] if len(url.path) > 1 else virtualhost
+        ssl = url.scheme == 'amqps' or ssl
```

Comment:
```python
ssl = url.scheme == 'amqps' or ssl or ssl_options
```

Reply:
What I did was intentional to keep the behaviour introduced in https://github.com/mosquito/aio-pika/commit/27b00b21366ab6f0fd09e93b1e0e3b223d884fcf#diff-1bac594ff175e0e7eeccd4a58e3f8269R387

My suggetion is to apply the change you proposed but also remove the filtering of `ssl_options` fields pass it as it is to the `connection_class`

This would let the `connection_class` crash and would give the user useful feedback instead of silently ignoring misspelled options

If you agree I'll edit the PR

Reply:
That part would be redundant. We already set ssl to True if `ssl_options` is not empty after including parameters from url. No point in checking it twice.

Reply:
When server's certificate required a client certificate, you have to pass the `ssl_options` parameter, containing certificate and private key paths.

The `ssl=True` or `amqps` schema handling is would be a nice and simple way to enables TLS, but it's should be has lower priority then `ssl_options`.

Reply:
> The ssl=True or amqps schema handling is would be a nice and simple way to enables TLS, but it's should be has lower priority then ssl_options.

Doesn't the PR do exactly that?
https://github.com/mosquito/aio-pika/pull/135/files#diff-1bac594ff175e0e7eeccd4a58e3f8269R414

Using  `amqps://` but not using a client certificate seems to be like a perfectly valid use case, but it's just extremely intuitive without this PR.